src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -44,20 +45,35 @@ internal object InstallCommand : Runnable {
                 }.install(Installer.Apk(apk, packageName))
             } catch (e: Exception) {
                 logger.severe(e.toString())
+                throw e
             }
 
             when (result) {
-                RootInstallerResult.FAILURE ->
+                RootInstallerResult.SUCCESS ->
+                    logger.info("Mounted the APK file")
+                RootInstallerResult.FAILURE -> {
                     logger.severe("Failed to mount the APK file")
-                is AdbInstallerResult.Failure ->
-                    logger.severe(result.exception.toString())
-                else ->
+                    throw Exception()
+                }
+                AdbInstallerResult.Success ->
                     logger.info("Installed the APK file")
+                is AdbInstallerResult.Failure -> {
+                    logger.severe(result.exception.toString())
```

Comment:
Why is the exception logged here but above a custom string?

Reply:
Sorry, not following.

Reply:
A couple of lines above, instead of result.exception, a failure string is logged, but not here, why?

Reply:
I forgot only AdbInstallerResult.Failure has an exception attribute. Might wanna do this:

```
logger.severe("Failed to install the APK file: ${result.exception}")
```